client/Add.js

```diff
@@ -0,0 +1,92 @@
+import { useState } from "react";
+
+
+function Add({ data, setVideoData })
+{
+    const [newTitle, setNewTitle] = useState("");
+    const [newURL, setNewURL] = useState("");
+
+    let newItem = {};
+
+
+    const addFunction = (data) => 
+    {
+        newItem.id = data.length;
+        newItem.title = newTitle;
+        newItem.url = newURL;
+        newItem.rating = 0;
+        newItem.added = new Date();
+

```

Comment:
if (!newItem.title || !newItem.url) return

Reply:
Wouldnt it be something like (newItem.title && newItem.url)?